examples/great_expectations_data_validation/run.py

```diff
@@ -12,8 +12,15 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
-
 from pipelines.validation import validation_pipeline
 
+from zenml.client import Client
+
 if __name__ == "__main__":
     validation_pipeline()
+
+    c = Client()
+    orchestrator = c.active_stack.orchestrator
+    orchestrator_config = orchestrator.config
+    if orchestrator_config.get("kubeflow_hostname"):
+        print("Kubeflow URL: `{orchestrator_config.kubeflow_hostname}`")
```

Comment:
Similarly we can use `experiment_tracker_url` to get the experiment tracker run 
![image](https://github.com/zenml-io/zenml/assets/4792082/693f1394-2e29-4cd7-8d82-4fde369f0c53)

Reply:
This I don't quite understand. The great expectations pipeline has no experiment tracker.